src/components/alphabet/images-space/ImagesSpace.vue

```diff
@@ -0,0 +1,13 @@
+<template>
+    <div class="images-space">
+        <div class="image-item" v-for="(uri, index) in getImagesUri()"
+            :key="index"
+            v-on:click="$emit('imageSelected', index)">
+            <img v-bind:src="uri"/>
+        </div>
+    </div>
+</template>
+
+<script src="./images-space.js"> </script>
```

Comment:
Rewrite all the components without js/css imports. Vuejs is all about keeping html/js/css in one .vue file